doc/plugins/org.polarsys.capella.preferences.doc/html/06. Preferences/6.2. Model preferences.html

```diff
@@ -135,12 +135,12 @@ <h2 id="Model">Model</h2>
 			<img height="633" width="874" border="0" src="Images/6.2.%20Model%20preferences_html_m2492a832.png"/>
 		</p>
 		<p>When &ldquo;
-			<b>Reuse of components</b>&rdquo; is activated, it activates to enable the reuse of components, creating several ï¿½Partsï¿½ from a Type. During a project creation, it activate a performance to reuse components.
+			<b>Reuse of components</b>&rdquo; is activated, it activates to enable the reuse of components, creating several �Parts� from a Type. During a project creation, it activate a performance to reuse components.
```

Comment:
can you look at these formatting, they looks weird

Reply:
I have generated the html files from mediawiki, in mediawiki I only changed, I will check some other mediawiki versions.

![image](https://user-images.githubusercontent.com/60220371/95096859-786cf600-0735-11eb-8bf4-c9c578a37c93.png)

Reply:
Done.